decisions.go

```diff
@@ -15,19 +15,21 @@ func decisions(nick, msg string) string {
 		return nick + ": Choose what? Usage: !choose choice1 or choice2"
 	}
 
-	return nick + ": The Powers that Be have chosen: " + choice // returning the choice
+	return fmt.Sprintf("%s: The powers that be have chosen: %s", nick, choice) // returning the choice
 }
 
 // the actual choose function
 func choose(msg string) string {
-	msg = strings.Trim(msg, "!choose ") // remove !choose trigger from the msg string
-	choices := strings.Split(msg, "or") // split on " or "
+	msg = strings.TrimPrefix(msg, "!choose ") // remove !choose trigger from the msg string
+	choices := strings.Split(msg, "or")       // split on " or "
```

Comment:
This will match on `fooorbarorbaz`. I think the intent is to match on `foo or bar or baz`. Here's an idea: why not split on commas?

`!choose foo, bar, baz`

This would allow the user to specify much longer strings and include the word `or`